projects/admin/src/app/record/detail-view/document-detail-view/document-detail-view.component.ts

```diff
@@ -115,12 +115,17 @@ export class DocumentDetailViewComponent implements DetailRecord, OnInit, OnDest
     if (rec.identifiedBy) {
       const queryParams = [];
       rec.identifiedBy.forEach((identifier: any) => {
-        queryParams.push(this._extractAndFormatQueryParams(identifier));
-        if (keysTypes.indexOf(identifier.type) > -1) {
-          const cidentifier = cloneDeep(identifier);
-          cidentifier.type = this.mappingtypes[cidentifier.type];
-          queryParams.push(this._extractAndFormatQueryParams(cidentifier));
-        }
+        if ((identifier.type == 'bf:Isbn') ||
+            (identifier.type == 'bf:Issn') ||
+            (identifier.type == 'bf:Doi') ||
+            (identifier.type == 'bf:Lccn') ) {
```

Comment:
`if (['bf:Isbn', 'bf:Issn', 'bf:Doi', 'bf:Lccn'].includes(identifier.type)) {`

Reply:
suggestion: `bf:IssnL` and `bf:Ean` ?

Reply:
Done. Thanks !